src/WindowManager.vala

```diff
@@ -157,6 +157,26 @@ namespace Gala {
                 });
             }
 #endif
+
+            var scroll_action = new SuperScrollAction (display);
+            scroll_action.triggered.connect (handle_super_scroll);
+
+            stage.add_action_full ("super-scroll-action", CAPTURE, scroll_action);
```

Comment:
Currently all of the stage stuff and keybindings etc. is done in show_stage maybe we should do this there as well?